drgn/commands/_builtin/crash/vm.py

```diff
@@ -0,0 +1,233 @@
+            try:
+                typ, val = _pid_or_task(arg)
+                if typ == "pid":
+                    pid_list.append(val)
+                elif typ == "task":
+                    task_list.append(val)
+            except (ValueError, CommandArgumentError):
+                invalid_args.append(arg)
+        for pid in pid_list:
+            code.append(f"    find_task(prog, {pid}),\n")
+        for addr in task_list:
+            code.append(f'    Object(prog, "struct task_struct *", value={addr:#x}),\n')
+
+        for arg in invalid_args:
+            code.append(f"    # vm: invalid task or pid value: {arg}\n")
+    code.append("]\n\n")
+
+    # Add processing logic
+    code.append(
+        """\
+for task in tasks:
+    # Get basic task info
+    pid = int(task.pid)
+    task_addr = task.value_()
+    cpu = task_cpu(task)
+    command = task.comm.string_().decode()
+
+    mm=task.mm
+    if mm:
+        # Get memory statistics
+        mm_addr = mm.value_()
+        pgd = mm.pgd.value_()
+        rss_info = get_task_rss_info(task)
+        rss = rss_info.total
+        total_vm = mm.total_vm.value_() * prog['PAGE_SIZE'].value_()
+        # Get VMA info
+        for vma in for_each_vma(mm):
+            vma_addr = vma.value_()
+            vm_start = vma.vm_start.value_()
+            vm_end = vma.vm_end.value_()
+            vm_flags = vma.vm_flags.value_()
+
+            # Get file path
+            if vma.vm_file:
+                try:
+                    file_path = inode_path(vma.vm_file.f_inode).decode()
+                    if not file_path.startswith("/"):
+                        file_path = "/" + file_path
+                except Exception:
+                    file_path = "<unknown>"
+            else:
+                file_path = "[anon]"
+    else:
+        # No memory mapping
+        mm_addr = 0
+        pgd = 0
+        rss = 0
+        total_vm = 0
+"""
+    )
+    code.print()
+
+
+def _print_task_vm_info(prog: Program, task: Object) -> None:
+    """Print virtual memory info for a single task"""
+    pid = int(task.pid)
+    cpu = task_cpu(task)
+    command = task.comm.string_().decode()
+    task_addr = task.value_()
+
+    print(f'PID: {pid}\tTASK: {task_addr:#x}\tCPU: {cpu}\tCOMMAND: "{command}"')
+
+    mm = task.mm
+    if not mm:
+        stats_rows = [["MM", "PGD", "RSS", "TOTAL_VM"], ["0", "0", "0k", "0k"]]
+        print_table(stats_rows)
+        print()
+        return
+
+    # Show memory statistics
+    rss_info = get_task_rss_info(task)
+    total_vm_kb = mm.total_vm.value_() * prog["PAGE_SIZE"].value_() // 1024
+
+    stats_rows = [
+        ["MM", "PGD", "RSS", "TOTAL_VM"],
+        [
+            f"{mm.value_():#x}",
+            f"{mm.pgd.value_():#x}",
+            f"{rss_info.total}k",
+            f"{total_vm_kb}k",
+        ],
+    ]
+    print_table(stats_rows)
+
+    # Show detailed VMA info
+    rows = [["VMA", "START", "END", "FLAGS", "FILE"]]
```

Comment:
Similar here, VMA, START, and END are centered, then FLAGS should be left-justified.